### ISSUE TITLE
Make sure you can update the address of the group. 

### ISSUE THREAD

**Issue by corsacca** [ChasmSolutions/zume-project-multilingual#13]
right now it gets randomized

**Comment by ChrisChasm** [comment]
Where is it randomized? Can you add a screenshot?

**Comment by corsacca** [comment]
![image](https://user-images.githubusercontent.com/24901539/34109710-bd1a1140-e404-11e7-904e-a2d76b3ffb1d.png)
I entered the letter "a" as the address. I saved and then it changed to an address in germany.
I know "a" is not an address. I'm guessing this is google doing it's best guess.
Maybe we can :
- save google's guess as a second field
- provide more visual feedback for address selection (like an amazon when set your shipping address)
I'm not sure google will be great at guessing addresses across the world.
